### ISSUE TITLE
请问chatGLM6b如何实现[mask]PVP模式的微调？

### ISSUE THREAD

**Issue by Pigtwo2** [THUDM/ChatGLM-6B#1341]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

请问chatGLM6b如何实现[mask]PVP模式的微调？

### Steps To Reproduce

请问chatGLM6b如何实现[mask]PVP模式的微调？

### Environment

```markdown
- OS:
- Python:
- Transformers:
- PyTorch:
- CUDA Support (`python -c "import torch; print(torch.cuda.is_available())"`) :
```
